src/main/java/org/apache/maven/plugins/assembly/AssemblerConfigurationSource.java

```diff
@@ -223,4 +223,12 @@ public interface AssemblerConfigurationSource {
      * @return Override group name.
      */
     String getOverrideGroupName();
+
+    /**
+     * @return mask which is applied to permissions of files/directories before they are put into assembly.
+     * If {@code null} then the mask is not explicitly configured and remains implementation-specific.
```

Comment:
Is it really implementation specific? Not just that there simply isn't a mask? How many implementations are there?

Reply:
Yes, it is. It is implementation-specific, because it depends on Plexus Archiver internals. Refer to that [comment](https://github.com/apache/maven-assembly-plugin/pull/148#discussion_r1254863967) for details, please.